consensus/types/src/fork_name.rs

```diff
@@ -201,6 +201,46 @@ impl ForkName {
     pub fn gloas_enabled(self) -> bool {
         self >= ForkName::Gloas
     }
+
+    pub fn fork_ascii(self) {
+        if self == ForkName::Fulu {
+            println!(
+                r#"
+                                  ╔═══════════════════════════════════════╗
+                                  ║                                       ║
+                                  ║   TO FULU, MOAR BLOBS TO ETHEREUM     ║
+                                  ║                                       ║
+                                  ║        III DECEMBER MMXXV             ║
+                                  ║                                       ║
+                                  ╚═══════════════════════════════════════╝
+            
+                =============================================================================
+                ||||                                                                     ||||
+                |---------------------------------------------------------------------------|
+                |___-----___-----___-----___-----___-----___-----___-----___-----___-----___|
+                / _ \===/ _ \   / _ \===/ _ \                   / _ \===/ _ \   / _ \===/ _ \
+               ( (.\ oOo /.) ) ( (.\ oOo /.) )                 ( (.\ oOo /.) ) ( (.\ oOo /.) )
+                \__/=====\__/   \__/=====\__/                   \__/=====\__/   \__/=====\__/
+                   |||||||         |||||||                         |||||||         |||||||
+                   |||||||         |||||||     \\/),               |||||||         |||||||
+                   |||||||         |||||||    ,'.' /,              |||||||         |||||||
+                   |||||||         |||||||   (_)- / /,             |||||||         |||||||
+                   |||||||         |||||||      /\_/ |__..--,  *   |||||||         |||||||
+                   |||||||         |||||||     (\___/\ \ \ / ).'   |||||||         |||||||
+                   |||||||         |||||||      \____/ / (_ //     |||||||         |||||||
+                   |||||||         |||||||       \\_ ,'--'\_(      |||||||         |||||||
+                   (oOoOo)         (oOoOo)       )_)_/ )_/ )_)     (oOoOo)         (oOoOo)
+                   J%%%%%L         J%%%%%L      (_(_.'(_.'(_.'     J%%%%%L         J%%%%%L
+                  ZZZZZZZZZ       ZZZZZZZZZ                       ZZZZZZZZZ       ZZZZZZZZZ
+                ===========================================================================
+              __|_________________________________________________________________________|__
```

Comment:
```suggestion
                |_________________________________________________________________________|
```
does it look better with a pyramid staircase?

Reply:
i made some minor changes to the stairs, i think they look better. lmk what you think